specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/virtualWan.json

```diff
@@ -7744,6 +7969,11 @@
             "$ref": "./network.json#/definitions/SubResource"
           },
           "description": "List of egress NatRules."
+        },
+        "dpdTimeoutSeconds": {
+          "type": "integer",
+          "format": "int32",
+          "description": "DPD timeout in seconds for VpnLink connection."
```

Comment:
>DPD [](http://example.com/codeflow?start=26&length=3)

It will be good to put full form of this in description.

Reply:
added full form in description